nvtabular/ops.py

```diff
@@ -1102,3 +1102,43 @@ def op_logic(self, gdf: cudf.DataFrame, target_columns: list, stats_context=None
             new_gdf[col] = self.f(gdf[col], gdf)
         new_gdf.columns = [f"{col}_{self._id}" for col in new_gdf.columns]
         return new_gdf
+
+
+class Filter(TransformOperator):
+    """
+    Enables to call Methods to cudf.Series. Returns a subset of the original gdf.
+
+    Parameters
+    -----------
+    f : lambda function
+        defines the function executed on dataframe level, expectation is lambda gdf: gdf[gdf.col...]
+        col is the cudf.Series defined by the context
+        gdf is the full cudf.DataFrame
```

Comment:
I think we could improve this comment - we don't need to talk about cudf.Series (since this Filter op could work on multiple series/columns) and the example given doesn't show how to filter any rows. How about:

```suggestion
    f : callable
        Defines a function that filter rows from a dataframe. As an example  ```lambda gdf: gdf[gdf.a >= 0]``` would 
        filter out all rows that had a negative value in the ```a``` column.
```